packages/web-frontend/src/containers/EnlargedDialog/EnlargedDialogContent.js

```diff
@@ -219,6 +219,10 @@ export class EnlargedDialogContent extends PureComponent {
     const { onSetDateRange, isLoading, viewContent, isExporting } = this.props;
     const { periodGranularity, startDate, endDate } = viewContent;
 
+    if (!periodGranularity) {
+      return null;
+    }
```

Comment:
Question: this makes a lot of sense, how come it wasn't an issue in the past? Or was it (does it fix an existing bug?)

Reply:
I believe that this is completely unneeded! The only place this is rendered is in this file (line 122) and states:
`{periodGranularity && this.renderPeriodSelector()}`